.gitignore

```diff
@@ -1 +1 @@
-data
\ No newline at end of file
+/data
```

Comment:
As it is, all folders called `data` are ignored, which includes `/internal/data/*`. I changed it to `/data` such that only the the `data` folder in the project root is ignored